test/system/README.md

```diff
@@ -82,7 +82,9 @@ the function or perhaps just a substring.
 Requirements
 ============
 
-The `jq` tool is needed for parsing JSON output.
+- The `jq` tool is needed for parsing JSON output.
+- The `bats` tool is needed for running tests.
+- The `skopeo` tool is needed for fetching remote images.
```

Comment:
There are more requirements, https://github.com/containers/podman/blob/c6f8de8a457800dfa3fb1fe7047f05f2fed34ebc/rpm/podman.spec#L142-L154C16

And if I learned one thing we are very bad at keeping these up to date. I think it would make more sense to just say the latest requirements are listed in the podman-tests rpm. We can add this command: `dnf repoquery --requires podman-tests`, this way we do not have to update the list here
cc @edsantiago

Reply:
I think what @Luap99 meant here was that you add something like "see rpm/podman.spec, under 'package test', for a list of requirements". Duplicating the list here is unmaintainable. Too late, though; we'll have to deal with that another day.

Nice work otherwise!